### ISSUE TITLE
revise "elongin complex" definition

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#18957]
GO:0070449    JSON
elongin complex
A transcription elongation factor complex that suppresses RNA polymerase II pausing, and may act by promoting proper alignment of the 3'-end of nascent transcripts with the polymerase catalytic site. Consists of a transcriptionally active Elongin A subunit (abut 100 kDa)and two smaller Elongin B (about 18 kDa) and Elongin C (about 15 kDa)subunits. PMID:12676794

@colinlog  can you suggest an update to cover initiation too?

**Comment by ValWood** [comment]
@colinlog  is elongin definitely also involved in initiation? I could not find any references to say so.

val

**Comment by colinlog** [comment]
PMC: 2042163 claims implication of TFIIS in initiation **and** elongation.

https://github.com/geneontology/go-ontology/issues/18837 is about the same protein domain / molecular function implication in termination of RNA polymerase III transcription -- via transcript cleavage stimulation. Such transcript cleavage has been referred to as an evolutionary conserved process that is performed in bacteria by GreA and GreB (ref: http://genesdev.cshlp.org/content/18/20/2437.full.pdf)


Danny Reinberg and colleagues (ref: http://genesdev.cshlp.org/content/18/20/2437.full.pdf) state that TFIIS is an elongation factor 'that promotes read through at pause sites' and refer to three papers: [... Fish and Kane 2002; Conaway et al.2003). TFIIS was initially identified by its ability to stimulate transcription in vitro (Natori et al. 1973),...

The human BLAST hit for yeast Dst1 is P23193, called TCEA1

But, this ticket is about the 'Elongin complex' isn't it?
That is made up of three subunits and it was reported in 1995 that it "increases the catalytic rate of transcribing RNA polymerase II by suppressing transient pausing by polymerase at multiple sites on DNA templates" Also work from the Conaway group, PMIDs: 7638163, 8654961 are shown in the uniprot entries.
   
Elongin A (uniprot: Q14241, ELOA, SIII p110) has a TFIIS domain, linking it to the above TFIIS blurb.
Elongin B (uniprot: Q15370 - ELOB, SIII, p18) has a ubiquitin-like domain) 
Elongin C (uniprot: Q15369 - ELOC, SIII,p15 ) has a SKP1 domain found in subunits of ubiquitin ligase complexes. ELOC together with ELOB binds to hydroxylproline in the HIF1A transcription factor to promote its degradation (crystal structure! PMID: 12004076) under normoxic conditions. ie; it is also a regulator of a transcription factor (see the ticket ??? about the need to distinguish terms for co-regulators that help TFs do their work at their target gene when they are DNA-bound, from regulators of TFs that control their access to DNA / DNA binding activity).

Elongin is discussed by  (ref: http://genesdev.cshlp.org/content/18/20/2437.full.pdf) as an elongation factor (explicitly not PIC formation nor initiation) whose ELOA subunit is transcriptionally active and whose ELOB and ELOC subunits are regulatory. They provide quite a few references @ValWood. And again, there appears to be experimental evidence that would implicate the TFIIS domain-bearing ELOA subunit (ELOA) in the repositioning of the 3;' end of the nascent RNA inside the RNA polymerase II catalytic site.

In summary, the same MF performed by the TFIIS domain that alters the conformation of RNA polymerase active sites can be annotated to initiation (human TCEA3, yeast Dst1) as TFIIS is demonstrated to be involved in initiation (PIC) and elongation processes. Elongin on the other had uses this MF conferred by the TFIIS domain in ELOA only in the elongation process. Furthermore, Elongin B and C do more than that, apparently, including VHL - HIFA1 regulation.

**Comment by ukemi** [comment]
Can you provide a new definition?

**Comment by ValWood** [comment]
Let's close this one. @colinlog  please reopen if you think the current definition is misleading.
